src/Slices/Agents/UI/Agents.test.tsx

```diff
@@ -9,9 +9,9 @@ import {
   QueryResolverImpl,
   getStoreInstance,
   CommandResolverImpl,
-  KeycloakAuthHelper,
   QueryManagerResolverImpl,
   CommandManagerResolverImpl,
+  defaultAuthContext,
```

Comment:
I don't get it, defaultAuthContext is just an object at this point, it's nothing more. Why aren't you using the actual context in the tests?

Reply:
Why don't you access the context directly in the concerned places instead? That's the whole point of having a context. 

You shouldn't need to do this in the CommandManagerResolverImpl 
<img width="225" alt="image" src="https://github.com/inmanta/web-console/assets/44098050/590c9e92-2262-4a7c-a04a-5d5b3a5ee762">

but access the authContext in the different managers if they require it. We go for a context to be able to avoid drilling down.

Reply:
Okay, make a ticket for it. But I had hoped that the refactor would allow the auth to be standalone and thus meant that we could remove the complexer logic relying on that.

Reply:
https://github.com/inmanta/web-console/issues/5840